virttest/utils_misc.py

```diff
@@ -621,16 +621,16 @@ def display_attributes(instance):
             LOG.debug("    %s: %s", name, value)
 
 
-def get_full_pci_id(pci_id):
+def get_full_pci_id(pci_id, session=None):
     """
     Get full PCI ID of pci_id.
 
     :param pci_id: PCI ID of a device.
     """
     cmd = "lspci -D | awk '/%s/ {print $1}'" % pci_id
     try:
-        return process.run(cmd, shell=True).stdout_text.strip()
-    except process.CmdError:
+        return cmd_status_output(cmd, shell=True).stdout_text.strip()
```

Comment:
It does not work with `AttributeError: 'tuple' object has no attribute 'stdout_text'`
https://github.com/avocado-framework/avocado-vt/blob/d688cc7f540dc52ca99ed55058314bb7d26bb073/virttest/utils_misc.py#L180
and https://github.com/avocado-framework/avocado-vt/blob/d688cc7f540dc52ca99ed55058314bb7d26bb073/virttest/utils_misc.py#L190-L191

Reply:
I have updated the code, and test is passed.

JOB LOG    : /root/avocado/job-results/job-2024-05-28T02.26-abb9a7e/job.log
 (1/1) Host_RHEL.m9.u5.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.9.4.0.x86_64.io-github-autotest-qemu.pktgen_perf.vp_vdpa_test_vm.q35: STARTED
 (1/1) Host_RHEL.m9.u5.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.9.4.0.x86_64.io-github-autotest-qemu.pktgen_perf.vp_vdpa_test_vm.q35: PASS (349.89 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /root/avocado/job-results/job-2024-05-28T02.26-abb9a7e/results.html
JOB TIME   : 351.14 s